framework/format/format.h

```diff
@@ -618,6 +619,14 @@ struct Dx12RuntimeInfoCommandHeader
     Dx12RuntimeInfo  runtime_info;
 };
 
+struct TlasToBlasDependencyHeader
```

Comment:
Would it be possible to make this a more generic dependency command, like ParentToChildDependencyHeader?  Can we determine dependency from the parent and child object types?  Or does this need to implicitly be specifically for AccelerationStructure handleIds?

Reply:
Although currently we don't need for other use cases, making it more generic sounds like a good idea.